phoenix-core/src/it/java/org/apache/phoenix/end2end/LikeExpressionIT.java

```diff
@@ -430,4 +430,26 @@ public void testParameterizedLikeExpression() throws Exception {
         rs = select.executeQuery();
         assertFalse(rs.next());
     }
+
+    @Test
+    public void testLikeExpression5173() throws Exception {
```

Comment:
please rename the test and instead add a comment pointing towards the JIRA instead.